### ISSUE TITLE
OneSignal fails Lighthouse tests

### ISSUE THREAD

**Issue by ayumihamsaki** [OneSignal/OneSignal-Website-SDK#360]
Hello,

Is it possible to update some of the OneSignal Code because it fails the Lighthouse tests for accessibility, see below screenshot:

![untitled](https://user-images.githubusercontent.com/17784082/40502836-159c8ff0-5f84-11e8-80b0-c7e0576b627d.png)

The two problems are as follows:

1. No alt-tag

This can be fixed by editing the file found here: https://github.com/OneSignal/OneSignal-Website-SDK/blob/master/src/popover/Popover.ts

Code line: 63

(Just add `alt="logo"`)

2. The default writing in the CSS doesn't pass WCAG 2.0 (AA) spec.

This can be fixed by editing the file found here: https://github.com/OneSignal/OneSignal-Website-SDK/blob/master/src/stylesheets/popover.scss

Change the CSS color and background from: `#4285f4` to a little bit darker blue such as: `#1165f1` and it will then pass WCAG 2.0 (AA) giving a ratio of 5.05

I am more than happy to create a Pull Request to make your software pass these simple things.

**Comment by jkasten2** [comment]
@ayumihamsaki Pull requests are welcome! The alt could be set to "notification icon" to be a bit more descriptive. I'll have to double check if the color was set to that specific value for any reason but this also sounds like a good change

**Comment by ayumihamsaki** [comment]
@jkasten2 Let me know what folder to send it to if not the master and I will create a pull request now with the above changes.

**Comment by jkasten2** [comment]
This is now live in the version [150400](https://github.com/OneSignal/OneSignal-Website-SDK/releases/tag/150400)!
